pkg/nanocbor/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME    = nanocbor
 PKG_URL     = https://github.com/bergzand/nanocbor
-PKG_VERSION = 97e8cbe943388dda0087ab703034fec268c7c12d
+PKG_VERSION = 57c48edaca2b5792a41119610035335761dd3c81
```

Comment:
This should not be necessary anymore with #16538 merged

Reply:
I will drop that commit when squashing/rebasing